src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const block = document.createElement('div');
```

Comment:
```suggestion
  const notification = document.createElement('div');
```
use descriptive variable names